internal/acctest/config/oauth/client/oauth_client_test.go

```diff
@@ -90,7 +90,7 @@ func TestAccOauthClient(t *testing.T) {
 		description:                        pointers.String("updatedDescription"),
 		logoUrl:                            pointers.String("https://example.com"),
 		redirectUris:                       []string{"https://example.com"},
-		allowAuthenticationApiInit:         pointers.Bool(false),
+		allowAuthenticationApiInit:         pointers.Bool(true),
```

Comment:
At some point we may want to add tests to cover expected errors such as errors from modifyplan

Reply:
I added a case for this internally.